plugin/src/main/java/net/dzikoysk/funnyguilds/config/serdes/VectorSerializer.java

```diff
@@ -0,0 +1,33 @@
+package net.dzikoysk.funnyguilds.config.serdes;
+
+import eu.okaeri.configs.schema.GenericsDeclaration;
+import eu.okaeri.configs.serdes.DeserializationData;
+import eu.okaeri.configs.serdes.ObjectSerializer;
+import eu.okaeri.configs.serdes.SerializationData;
+import org.bukkit.util.Vector;
+import org.jetbrains.annotations.NotNull;
+
+public class VectorSerializer implements ObjectSerializer<Vector> {
```

Comment:
Maybe add this to okaeri-configs/serdes-bukkit ( ͡° ͜ʖ ͡°)

Reply:
soon:tm:

Reply:
Added in https://github.com/OkaeriPoland/okaeri-configs/commit/55647fe565e15fcef86f0b0e95741934f2181ea0. However, it is worth nothing that FunnyGuilds isn't currently using `serdes-bukkit` and even has its own, custom MaterialTransformer. Realistically, such change should be done not here, but in a bigger configuration refactor.